presto-kudu/pom.xml

```diff
@@ -172,5 +172,25 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
```

Comment:
What is this? It is not used anywhere here, am I missing something?

Reply:
The Kudu client library uses a new annotation library in the version 1.8.0. These annotations from Apache Yetus provide "the means to document a well defined supported interface for downstream projects". As a side effect, this new dependency breaks the javadoc build. The dependency is only optional, but seems to be required for the javadoc build. Adding it explicitly to the profile which is used for the javadoc build resolved the build problem.

Reply:
So it should not be named ci, but rather `javadoc`. 

Please add a comment.

If this is breaking javadoc build always, then please put it into regular build (remove the profile).

Reply:
done